crates/matrix-sdk/src/widget/mod.rs

```diff
@@ -197,3 +204,11 @@ impl WidgetDriver {
         Ok(())
     }
 }
+
+// TODO: Decide which module this type should live in
```

Comment:
I'd put it in `actions`, since it's a type used and appearing in `ReadStateEventCommand` for the first time.

Reply:
Well, it's likely also going to be used elsewhere soon.